pulp_2_tests/tests/rpm/api_v2/test_iso_sync_publish.py

```diff
@@ -171,3 +173,31 @@ def test_all(self):  # pylint:disable=no-self-use
         path = urljoin(urljoin('/pulp/isos/', repo['id'] + '/'), iso_name)
         iso2 = client.get(path).content
         self.assertEqual(iso, iso2)
+
+
+class SyncInvalidFileTestCase(unittest.TestCase):
+    """Perform sync on an invalid files repository.
```

Comment:
Should the expected behavior be added as well? This tells us what the test does but not what the outcome should be